vmdb/lib/workers/event_catcher_kubernetes.rb

```diff
@@ -38,8 +38,35 @@ def monitor_events
   end
 
   def process_event(event)
-    # TODO: some sane event filtering
-    $log.info "#{log_prefix} Caught event [#{event}]"
-    # TODO: add_queue kubernetes event
+    event_data = {
+      :timestamp => event.object.lastTimestamp,
+      :kind      => event.object.involvedObject.kind,
+      :name      => event.object.involvedObject.name,
+      :namespace => event.object.involvedObject['table'][:namespace],
+      :reason    => event.object.reason,
+      :message   => event.object.message
+    }
+
+    @enabled_events ||= VMDB::Config.new("vmdb").config.fetch_path(
+      :workers, :worker_base, :event_catcher, :event_catcher_kubernetes,
+      :enabled_events)
```

Comment:
You can get this from `worker_settings` like in the [event_catcher_openstack.rb](https://github.com/simon3z/manageiq/blob/events-timeline/vmdb/lib/workers/mixins/event_catcher_openstack_mixin.rb#L13-L15).